diff --git a/node_modules/html-to-image/es/dataurl.js b/node_modules/html-to-image/es/dataurl.js
index 174e4b9..f55ef70 100644
--- a/node_modules/html-to-image/es/dataurl.js
+++ b/node_modules/html-to-image/es/dataurl.js
@@ -8,7 +8,7 @@ export function makeDataUrl(content, mimeType) {
     return `data:${mimeType};base64,${content}`;
 }
 export async function fetchAsDataURL(url, init, process) {
-    const res = await fetch(url, init);
+    throw Error('fetch call was blocked by AC');
     if (res.status === 404) {
         throw new Error(`Resource "${res.url}" not found`);
     }
diff --git a/node_modules/html-to-image/es/embed-webfonts.js b/node_modules/html-to-image/es/embed-webfonts.js
index e6ddae4..4d5363c 100644
--- a/node_modules/html-to-image/es/embed-webfonts.js
+++ b/node_modules/html-to-image/es/embed-webfonts.js
@@ -7,7 +7,7 @@ async function fetchCSS(url) {
     if (cache != null) {
         return cache;
     }
-    const res = await fetch(url);
+    throw Error('fetch call was blocked by AC');
     const cssText = await res.text();
     cache = { url, cssText };
     cssFetchCache[url] = cache;
